src/content/en/2020/seo.md

```diff
@@ -9,18 +8,18 @@ reviewers: [natedame, catalinred, dsottimano, dwsmart, en3r0, ibnesayeed, bazzad
 analysts: [Tiggerito, antoineeripret]
 translators: []
 aleyda_bio: SEO consultant, author, speaker and entrepreneur. Founder of <a href="https://www.orainti.com/">Orainti</a> (a boutique SEO consultancy working with some of the top Web properties and brands, from SaaS to marketplaces) and co-founder of <a href="https://remoters.net/">Remoters.net</a> (a free remote work hub, featuring remote jobs, tools, events, guides, and more to facilitate remote work). Aleyda enjoys sharing about SEO through her <a href="https://www.aleydasolis.com/en/blog/">blog</a>, the <a href="https://www.aleydasolis.com/en/seo-tips/">#SEOFOMO newsletter</a>, the <a href="https://www.aleydasolis.com/en/crawling-mondays-videos/">Crawling Mondays</a> video and podcast series and over <a href="https://twitter.com/aleyda">Twitter</a>.
-ipullrank_bio: An artist and a technologist, all rolled into one, Michael King is the founder of enterprise digital marketing agency, <A href="https://ipullrank.com">iPullRank</a> and founder of Natural Language Generation app <a href="https://www.copyscience.io">CopyScience</a>. Effortlessly leaning on his background as an independent hip-hop musician, Mr. King is a compelling content creator and an award-winning dynamic speaker who is called upon to contribute to technology and marketing conferences and blogs all over the world. You can find Mike talking trash on <a href="https://twitter.com/ipullrank">Twitter</a>
-#fellowhuman1101_bio: TODO
+ipullrank_bio: An artist and a technologist, all rolled into one, Michael King is the founder of enterprise digital marketing agency, <a href="https://ipullrank.com">iPullRank</a> and founder of Natural Language Generation app <a href="https://www.copyscience.io">CopyScience</a>. Effortlessly leaning on his background as an independent hip-hop musician, Mr. King is a compelling content creator and an award-winning dynamic speaker who is called upon to contribute to technology and marketing conferences and blogs all over the world. You can find Mike talking trash on <a href="https://twitter.com/ipullrank">Twitter</a>
+#fellowhuman1101_bio: TODO.
 discuss: 2043
 results: https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/
 queries: 07_SEO
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
+#featured_quote: TODO.
+#featured_stat_1: TODO.
+#featured_stat_label_1: TODO.
+#featured_stat_2: TODO.
+#featured_stat_label_2: TODO.
+#featured_stat_3: TODO.
+#featured_stat_label_3: TODO.
```

Comment:
@aleyda can you suggest featured quote/stats?

Reply:
@rviscomi In process to confirm with @ipullrank and @fellowhuman1101 the [ones here](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1589#discussion_r532228367) and [here](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1589#discussion_r533993766). As soon as I have confirmation I'll add :)

Reply:
```suggestion
#featured_quote: Despite the growing use of mobile devices and Google’s move to a Mobile first index, non-trivial disparities were found across mobile vs. desktop pages, like the one between mobile and desktop links.
#featured_stat_1: Mobile home pages are not including the viewport tag
#featured_stat_label_1: 10.84%
#featured_stat_2: Mobile websites scored 'Good' Core Web Vitals
#featured_stat_label_2: 19.96%
#featured_stat_3: More words are displayed in the rendered than the raw HTML of the median mobile site
#featured_stat_label_3: 11.5%
```

Reply:
@rviscomi ouch! I just saw it. I updated it :) Thanks for the heads up.